Assets/Scripts/Game/Formulas/FormulaHelper.cs

```diff
@@ -2340,11 +2357,8 @@ public static int CalculateCastingCost(SpellRecord.SpellRecordData spell)
             byte[] rangeTypeModifiers = { 2, 2, 3, 4, 5 };
 
             int cost = 0;
-            int skill = 50;  // 50 is used for item enchantments, which is all this spell is used for now.
-                             // This function could be used as in classic to get casting costs for spells
-                             // cast by entities by replacing skillUsedForEnchantedItemCost with the skill
-                             // of the caster in the effect's spell school.
-
+            int skill = 50; // 50 is used for item enchantments
```

Comment:
When I was working on reversing this I found that it actually uses the player's skill in classic. Elenwel confirmed this here: https://forums.dfworkshop.net/viewtopic.php?f=23&t=3621&sid=a1aa5bf0451e688b46ee586f57093325#p43434

It makes more sense to me to have it use a constant skill value for enchanted items but if we're going for parity with classic it should use the player's skill.

Reply:
It uses the payer skill, see the code I added below. The 50 constant is only kept for the Magic item maker. :)

Reply:
That'll teach me to code review while distracted. Great work!